src/Console/Command/Git/InitCommand.php

```diff
@@ -157,11 +157,16 @@ protected function generateHookCommand(string $command): string
         $process = $this->processBuilder->buildProcess($arguments);
 
         // Make the grumphp command relative to path where the config is stored (if it uses that executable)
-        return str_replace(
-            $this->filesystem->ensureUnixPath($this->paths->getProjectDir()),
-            '.',
-            $process->getCommandLine()
-        );
+        if (0 ===
+            strpos($process->getCommandLine(), $this->filesystem->ensureUnixPath($this->paths->getProjectDir()))) {
```

Comment:
I'm not really sure what you mean by `the getCommandLine() escapes the parameters with ' or "`. 

Also i'm not sure what `$process->getCommandLine()` is supposed to return. I assumed it always was an absolute path, since the process of this function is making it relative.
That being said, the only valid way to make it relative, is when the project dir shares the same absolute path.
If you intention is to return a relative path by returning everything *after* the project dir, this code should be changed to some regex or strpos magic, to return exactly that.